app/services/agenda-service.js

```diff
@@ -267,5 +267,31 @@ export default Service.extend(ModifiedMixin, isAuthenticatedMixin, {
         type: 'error'
       }));
     }
+  },
+
+  async retrieveModifiedDateFromNota(agendaItem) {
+    const newsletterInfoForSubcase = await agendaItem.get('subcase.newsletterInfo');
+    const documents = await agendaItem.get('subcase.documents');
+    if (!documents) {
+      return null;
+    }
+
+    const nota = await agendaItem.get('nota');
+    if(!nota) {
+      return null;
+    }
+
+    const documentVersion = await nota.get('lastDocumentVersion');
+    const modifiedDateFromMostRecentlyAddedNotaDocumentVersion = documentVersion.modified;
```

Comment:
Niet helemaal mee waarom we de modified datum gebruiken van de nota en niet de datum van creatie.
Dit als we ervan uitgaan dat de gebruikers enkel die melding willen krijgen als er een nieuwe versie is.
Modified wordt ook aangepast als de naam gewijzigd wordt of het toegangsniveau.
In die gevallen hoeft die melding niet te tonen denk ik